scripts/tests/test_generate_sitemap.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+import shutil, tempfile, xml, os
+from urlparse import urljoin
+from website import settings
+from scripts.generate_sitemap import main
+from osf_tests.factories import AuthUserFactory, PreprintProviderFactory, PreprintFactory, ProjectFactory, \
+    RegistrationFactory, CollectionFactory
+
+@pytest.mark.django_db
+class TestGenerateSitemap:
```

Comment:
use pytest.fixture() to create fixtures for reusable components (look through api_tests/nodes/views files for examples)